docs/router/configuration.mdx

```diff
@@ -1038,6 +1038,7 @@ These rules apply to requests being made from the Router to all Subgraphs.
 | Environment Variable                       | YAML                                       | Required                                      | Description                                                                                                                                                                                            | Default Value |
 | ------------------------------------------ | ------------------------------------------ | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
 |                                            | persisted_operations                       | <Icon icon="square" />                        | The configuration for the persisted operations.                                                                                                                                                        |               |
+| PERSISTED_OPERATIONS_DISABLED              | persisted_operations.disabled              | <Icon icon="square" />                        | When set to `true`, all requests using persisted operations will be blocked and the router will return an error response.                                                                              | false         |
```

Comment:
I think we also always include the fields in the example yaml

Reply:
fixed